include/cufinufft/utils.h

```diff
@@ -9,13 +9,22 @@
 
 #include <cuda_runtime.h>
 
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#include <windows.h>
+#else
 #include <sys/time.h>
```

Comment:
We should just use chrono as per CPU version.

Reply:
Yes, it should share the timer implementation with finufft.